legend-sdlc-server/src/main/resources/docker/config/config.json

```diff
@@ -0,0 +1,106 @@
+{
+  "applicationName": "Legend SDLC",
+  "server": {
+    "adminContextPath": "/admin",
+    "applicationConnectors": [
+      {
+        "type": "http",
+        "port": ${SDLC_PORT},
+        "maxRequestHeaderSize": "128KiB"
+      }
+    ],
+    "adminConnectors": [
+      {
+        "type": "http",
+        "port": 7071
+      }
+    ],
+    "gzip": {
+      "includedMethods": [
+        "GET",
+        "POST"
+      ]
+    },
+    "requestLog": {
+      "type": "classic",
+      "level": "OFF",
+      "appenders": [
+        {
+          "type": "console",
+          "logFormat": "OFF"
+        }
+      ]
+    },
+    "rootPath": "/api"
+  },
+  "projectStructure": {
+    "projectCreation": {
+      "groupIdPattern": "^org\\.finos\\.legend\\..+"
+    },
+    "extensionProvider": {
+      "org.finos.legend.sdlc.server.gitlab.finos.FinosGitlabProjectStructureExtensionProvider": {}
+    }
+  },
+  "filterPriorities": {
+    "GitLab": 1,
+    "org.pac4j.j2e.filter.CallbackFilter": 2,
+    "org.pac4j.j2e.filter.SecurityFilter": 3,
+    "CORS": 4
+  },
+  "pac4j": {
+    "callbackPrefix": "/api/pac4j/login",
+    "mongoUri": "${MONGODB_URI}",
+    "mongoDb": "${MONGODB_NAME}",
+    "clients": [
+      {
+        "org.finos.legend.server.pac4j.gitlab.GitlabClient": {
+          "name": "gitlab",
+          "clientId": "${GITLAB_APP_ID}",
+          "secret": "${GITLAB_APP_SECRET}",
+          "discoveryUri": "https://${GITLAB_HOST}/.well-known/openid-configuration",
+          "scope": "openid profile api"
+        }
+      }
+    ],
+    "mongoSession": {
+      "enabled": "${MONGO_SESSION_ENABLED}",
+      "collection": "userSessions"
+    },
+    "bypassPaths": [
+      "/api/info"
+    ]
+  },
+  "features": {
+    "canCreateProject": true,
+    "canCreateVersion": true
+  },
+  "gitLab": {
+    "newProjectVisibility": "public",
+    "projectTag": "pure",
+    "projectIdPrefix": "UAT",
```

Comment:
These don't seem like appropriate default values, almost regardless of the target users.

Reply:
Good catch. I'll change the "pure" tag to "legend".   Does the Studio app treat "UAT" as a special value ?

Reply:
You don't have to have a projectIdPrefix. I'm not sure one would be particularly meaningful in this case. It's mostly there for cases where there are multiple SDLC Servers running to ensure there's no id conflict.

"legend" is definitely a better tag.